core/src/stores/postgres.rs

```diff
@@ -90,38 +108,38 @@ impl PostgresStore {
         if let Some(filter) = filter {
             if let Some(tags_filter) = &filter.tags {
                 if let Some(tags) = &tags_filter.is_in {
-                    where_clauses.push(format!("tags_array && ${}", p_idx));
+                    where_clauses.push(format!("{}tags_array && ${}", prefix.as_str(), p_idx));
                     params.push(tags as &(dyn ToSql + Sync));
                     p_idx += 1;
                 }
                 if let Some(tags) = &tags_filter.is_not {
-                    where_clauses.push(format!("NOT tags_array && ${}", p_idx));
+                    where_clauses.push(format!("NOT {}tags_array && ${}", prefix.as_str(), p_idx));
                     params.push(tags as &(dyn ToSql + Sync));
                     p_idx += 1;
                 }
             }
 
             if let Some(parents_filter) = &filter.parents {
                 if let Some(parents) = &parents_filter.is_in {
-                    where_clauses.push(format!("parents && ${}", p_idx));
+                    where_clauses.push(format!("dsn.parents && ${}", p_idx));
                     params.push(parents as &(dyn ToSql + Sync));
                     p_idx += 1;
                 }
                 if let Some(parents) = &parents_filter.is_not {
-                    where_clauses.push(format!("NOT parents && ${}", p_idx));
+                    where_clauses.push(format!("NOT dsn.parents && ${}", p_idx));
```

Comment:
This is quite brittle isn't it. Any tweak to the query will break I think we need to have that an argument no?